clang/lib/Lex/LiteralSupport.cpp

```diff
@@ -1117,6 +1118,26 @@ NumericLiteralParser::NumericLiteralParser(StringRef TokSpelling,
       if (isImaginary) break;   // Cannot be repeated.
       isImaginary = true;
       continue;  // Success.
+    case '_':
+      if (isFPConstant)
+        break; // Invalid for floats
+      if (HasSize)
+        break;
+      if (PossibleBitInt)
+        break; // Cannot be repeated.
+      if (LangOpts.CPlusPlus && s + 1 < ThisTokEnd && s[1] == '_') {
+        // Scan ahead to find possible rest of BitInt suffix
+        for (const char *c = s; c != ThisTokEnd; ++c) {
+          if (*c == 'w' || *c == 'W') {
```

Comment:
I think we're probably better off consuming all of the __wb/__uwb here rather than falling through to the code on ~1151.

Reply:
Should we allow cases like __iwb or __wbj? Or do we only allow the unsigned modifier?

Reply:
We definitely should not allow `__iwb` because that puts the `i` suffix in the middle of the `__wb` suffix. However, because we accept `iwb` or `wbi` in C, I think it's reasonable to support `i__wb` or `__wbi` in C++: https://godbolt.org/z/nb9d9qWKK